pandas/tests/io/test_pickle.py

```diff
@@ -456,42 +456,10 @@ def mock_urlopen_read(*args, **kwargs):
         tm.assert_frame_equal(df, result)
 
 
-@td.skip_if_no("gcsfs")
-@pytest.mark.parametrize("mockurl", ["gs://gcs.com", "gcs://gcs.com"])
-def test_pickle_gcsurl_roundtrip(monkeypatch, mockurl):
-    with tm.ensure_clean() as path:
-
-        class MockGCSFileSystem:
-            def __init__(self, *args, **kwargs):
-                pass
-
-            def open(self, *args):
-                mode = args[1] or None
-                f = open(path, mode)
-                return f
-
-        monkeypatch.setattr("gcsfs.GCSFileSystem", MockGCSFileSystem)
-        df = tm.makeDataFrame()
-        df.to_pickle(mockurl)
-        result = pd.read_pickle(mockurl)
-        tm.assert_frame_equal(df, result)
-
-
-@td.skip_if_no("s3fs")
-@pytest.mark.parametrize("mockurl", ["s3://s3.com", "s3n://s3.com", "s3a://s3.com"])
-def test_pickle_s3url_roundtrip(monkeypatch, mockurl):
-    with tm.ensure_clean() as path:
-
-        class MockS3FileSystem:
-            def __init__(self, *args, **kwargs):
-                pass
-
-            def open(self, *args):
-                mode = args[1] or None
-                f = open(path, mode)
-                return f
-
-        monkeypatch.setattr("s3fs.S3FileSystem", MockS3FileSystem)
+@td.skip_if_no("fsspec")
```

Comment:
I think we'll still want the old tests.

Reply:
@TomAugspurger is this one still relevant, or can it be "resolved" ?